vsc_pdf_template_transformer/lib/models/tpl_repeater.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+import 'package:vsc_pdf_template_transformer/utils/widget_json_converter.dart';
+import '../utils/widget_builder.dart' as wb;
+
+part 'tpl_repeater.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplRepeater {
+  String className = 'TplRepeater';
+
+  @WidgetJsonConverter()
+  List<dynamic>? array;
+  @WidgetJsonConverter()
+  wb.WidgetBuilder? childTemplate;
+
+  TplRepeater();
+
+  factory TplRepeater.fromJson(Map<String, dynamic> json) =>
+      _$TplRepeaterFromJson(json);
+
+  Map<String, dynamic> toJson() => _$TplRepeaterToJson(this);
+
+  List<Widget?> toPdf(Map<String, dynamic> data) {
```

Comment:
You will probably need a common method for other widgets that take a list of children (TplColumn, TplRow, TplMultiPage, etc.) so that they can detect if there is a TplRepeater widget in the list and call this method to add the resulting list of widgets to the array of children. Repeater generates child widgets, so they have to be added somehow. The task has an example of a repeater embedded in a Column. You can use the sample_invoice.json data to test it.

This also should return `List<Widget>` rather than `List<Widget?>`